cmd/tracee-ebpf/flags/filter.go

```diff
@@ -288,11 +295,10 @@ func PrepareFilter(filtersArr []string) (tracee.Filter, error) {
 	return filter, nil
 }
 
-func prepareEventsToTrace(eventFilter *filters.StringFilter, setFilter *filters.StringFilter, eventsNameToID map[string]events.ID) ([]events.ID, error) {
-	eventFilter.Enable()
```

Comment:
I understand that this was not used anywhere, correct?

Reply:
Correct.